src/subdomains/voting/application/services/voting.service.ts

```diff
@@ -72,6 +72,24 @@ export class VotingService implements OnModuleInit {
     }
   }
 
+  async getAllData(): Promise<CfpAllData> {
+    const voteResult = await this.getCurrentVotes();
+    const voterCount = await this.masternodeService.getAllVoters().then((l) => l.length);
+    const cfpInfos = await this.getCurrentCfpList().then((c) =>
+      c.map((cfp) => ({
+        number: cfp.number,
+        title: cfp.title,
+        lockVotes: cfp.lockVotes,
+        endDate: cfp.endDate,
+        endHeight: cfp.endHeight,
+        type: cfp.type,
+        status: cfp.status,
+      })),
```

Comment:
Didn't we want to add the info, if LOCK has already voted on the respective proposal?